CMakeLists.txt

```diff
@@ -44,26 +45,34 @@ add_subdirectory(external)
 
 
 # Find the QtWidgets library
+set(QT_COMPONENTS
+  Core
+  Widgets
+  Gui
+  OpenGL
+)
 find_package(Qt6 QUIET
-  COMPONENTS
-   Core
-   Widgets
-   Gui
-   OpenGL
+  COMPONENTS ${QT_COMPONENTS}
 )
```

Comment:
Why do you force `Qt6`? The `QT_VERSION` loses its meaning. Suppose I have `Qt6` installed in parallel with some other version, but I want to use the other one exactly. How do I achieve that? In that context, the `QT_VERSION` variable becomes misleading.

After the comment above, I dived deeper into the PR. As it seems to be, the `QT_VERSION` is not actually the `QT_VERSION`, but the `Qt5` subversion just in case `Qt6` was not found. IMHO the variable name isn't good for that application.

Reply:
Please have a look at the latest master.

In order to specify the Qt version we can use cmake tools, I reperat my old comment here::


It is usually done by defining an enviroment variable `Qt5_DIR`.
It could be, for example: `Qt5_DIR=<QTDIR>/lib/cmake/Qt5`

Alternatively you could pass the root directory of your Qt installation when configuring the cmake.
For example

```
cmake ..